src/extras/ShapeUtils.js

```diff
@@ -58,20 +58,22 @@ class ShapeUtils {
 
 		//
 
-		let holeIndex = contour.length;
+		let holeFirstVertIndex = contour.length;
```

Comment:
As mentioned in https://github.com/mrdoob/three.js/pull/30698#issuecomment-2731449980, please revert all variable name changes and comments modifications.